packages/nimble-components/src/table-column/date-text/tests/table-column-date-text.spec.ts

```diff
@@ -498,6 +477,38 @@ describe('TableColumnDateText', () => {
     });
 
     describe('with static config', () => {
+        // prettier-ignore
+        async function setupWithConfig(): Promise<Fixture<Table<SimpleTableRecord>>> {
+            return fixture<Table<SimpleTableRecord>>(
+                html`<nimble-table style="width: 700px">
+                        <${tableColumnDateTextTag} field-name="field" group-index="0"
+                            format="custom"
+                            custom-locale-matcher="lookup"
+                            custom-weekday="short"
+                            custom-era="short"
+                            custom-year="2-digit"
+                            custom-month="short"
+                            custom-day="2-digit"
+                            custom-hour="2-digit"
+                            custom-minute="2-digit"
+                            custom-second="2-digit"
+                            custom-time-zone-name="longOffset"
+                            custom-format-matcher="basic"
+                            custom-hour12="false"
+                            custom-time-zone="America/Chicago"
+                            custom-calendar="hebrew"
+                            custom-day-period="short"
+                            custom-numbering-system="fullwide"
+                            custom-date-style="long"
+                            custom-time-style="long"
+                            custom-hour-cycle="h23"
+                            >
```

Comment:
I think this should be out-dented one level.
```suggestion
                        >
```